### ISSUE TITLE
Fix `custom-property-no-missing-var-function` false negatives for fallback values within `var()`

### ISSUE THREAD

**Issue by src-code** [stylelint/stylelint#8099]
### What minimal example or steps are needed to reproduce the bug?

```css
height: var(--primary-height, --fallback-height);
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "custom-property-no-missing-var-function": true
  }
}
```

### How did you run Stylelint?

```shell
stylelint “**/*.css” —config myconfig.json
```

### Which Stylelint-related dependencies are you using?

```json
{
        "stylelint": "^16.10.0",
        "stylelint-config-html": "^1.1.0",
        "stylelint-config-prettier-scss": "^1.0.0",
        "stylelint-config-standard": "^36.0.1",
        "stylelint-config-standard-scss": "^13.1.0",
        "stylelint-declaration-strict-value": "^1.10.6",
}
```

### What did you expect to happen?

Rule should have flagged the missing `var()` syntax on the fallback value.  The correct syntax would be:

```css
height: var(--primary-height, var(--fallback-height));
```

### What actually happened?

The test passed despite the missing `var()`

**Comment by ryo-manba** [comment]
I confirmed the behavior using the [Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcCBOEIBcAEwA6A7DGAtIQBYCM8GZADNQA4AeA3HgcSQEyUCs9zeAXzx4AhllYYSMAJYBzEmkrsyLfJJnzFGAG4ikACmUAaIqQ4BKVQSlyFlXQeM69hs+cuC8IIyABm0gBsYADkRAFs4RBgGcLoggDowAGck73AIXH9ZBBAcNWwQJABXIKSCyjyCAgKwIqS0CDDCOhQ6GCQ0AE9CXAhCMOkU6VxZQgdCXyLcMDRpDPKMNGKYCSFcATTITLkAMQgkMJE0HIArJIy02DpUxEqMAvrOoIDhtAWCgKOYeoKjCQeus9XoQtllCPURLgoHooO8QJ80N83iBBCABEA).

As in the [example on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/var#syntax), it is common to use a fallback value with `var`, so it might be worth supporting this case.

If a fallback is specified, checking each element in `node.nodes` (excluding commas) could potentially address this issue.
https://github.com/stylelint/stylelint/blob/030691951cda399b22e62e605f6b2201bc03ce84/lib/rules/custom-property-no-missing-var-function/index.mjs#L65-L70
